### PR TITLE
fix central jira actions

### LINKED ISSUE #44: [Bug] Labels do not sync to Jira on issue creation
Issue is created but the labels fail to sync.

Relevant workflow: https://github.com/dbt-labs/actions/blob/main/.github/workflows/jira-creation.yml

### PR DESCRIPTION
**Pull request by emmyoop**
resolves #44 

- Add/update versions on all jira actions
- Correct `$GITHUB_STATE` -> `$GITHUB_OUTPUT`

### CODE REVIEW

File: .github/workflows/jira-creation.yml [hunk 1]
```diff
@@ -104,14 +104,14 @@ jobs:
...
-          echo "Running for label:   ${{ matrix.label }}""
+          echo "Running for label:   ${{ matrix.label }}"
           echo "Issue ID:            ${{ needs.create-issue.outputs.issueId }}"
 
       - name: Setup Jira
         uses: atlassian/gajira-cli@v2.0.2
 
       - name: Login
-        uses: atlassian/gajira-login@v2.0.0
+        uses: atlassian/gajira-login@v2.0.1
```
Comment (leahwicz) [inline .github/workflows/jira-creation.yml:114]:
```suggestion
        uses: atlassian/gajira-login@v2.0.2
```
Line 46 changes the version to 2.0.2 so making this the same for consistency

Comment (leahwicz) [reply .github/workflows/jira-creation.yml:114]: Or is the correct version 2.0.1 because I see you use that in 2 places now

**Review by leahwicz — APPROVED** [review]
LGTM other than that login Action's version mismatch

File: .github/workflows/jira-creation.yml (hunk 1 above)
Comment (emmyoop) [reply .github/workflows/jira-creation.yml]: Good catch.  That was meant for the `gajira-create` version.  Fixed `gajira-login` to point to 2.0.0. everywhere.

**Pull request merged**